apps/api/models.py

```diff
@@ -102,24 +105,50 @@ class APIKey(models.Model):
     class Meta:
         db_table = 'api_key'
 
+    def __unicode__(self):
+        return (
+            u'<{cls} user={user}, type={type}, key={key} secret=...>'
+            .format(cls=self.__class__.__name__, key=self.key,
+                    type=self.type, user=self.user))
+
     @classmethod
     def get_jwt_key(cls, **query):
```

Comment:
This does feel like a place that we might find a bug. Could you default them both to `None` and throw before performing the query if we don't have values? Checking against `None` might not be the best. Maybe check if they're both falsey would be best.

```py
@classmethod
def get_jwt_key(cls, user=None, key=None, **query):
    if not user and not key:
        raise ValueError('You must provide a user or a key')
    ...